tensorrt_llm/_torch/pyexecutor/model_engine.py

```
@@ -3123,6 +3124,31 @@ def _get_lora_params_from_requests(self,
 
         return lora_params
 
+    def _prepare_ulysses_context_parallel_inputs(
+            self, inputs: Dict[str, Any], gather_ids: Optional[torch.Tensor]):
+        """
+            input_ids: [c0, ctx1, ..., ctxn, gen0, gen1, ..., genm]
+        """
+        # TODO 加文档，全切，建议用PD分离仅P使用
```

Ruff (0.14.10)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)